services/slam/builtin/builtin.go

```diff
@@ -32,11 +32,11 @@ import (
 	"go.viam.com/rdk/rimage/transform"
 	"go.viam.com/rdk/services/slam"
 	"go.viam.com/rdk/services/slam/grpchelper"
-	"go.viam.com/rdk/spatialmath"
```

Comment:
[note] not sure why these changed...

Reply:
My guess is the linter ignores aliases when ordering them, so it's treating it as

```
"go.viam.com/rdk/services/slam/grpchelper"
"go.viam.com/rdk/services/slam/slam_copy/config"
"go.viam.com/rdk/services/slam/slam_copy/dataprocess"
"go.viam.com/rdk/services/slam/slam_copy/utils"
"go.viam.com/rdk/spatialmath"
"go.viam.com/rdk/utils"
```

which makes sense.

I'm not sure why it didn't make these changes in Nick's PR to add slam_copy, though.